src/platform/packages/shared/shared-ux/chrome/navigation/src/ui/components/panel/panel_nav_item.tsx

```diff
@@ -55,22 +57,35 @@ const panelNavStyles = ({ euiTheme }: Theme) => css`
 `;
 
 export const PanelNavItem: FC<Props> = ({ item }) => {
-  const { navigateToUrl } = useServices();
+  const { navigateToUrl, eventTracker, basePath } = useServices();
   const { activeNodes } = useNavigation();
   const { close: closePanel } = usePanel();
-  const { id, icon, deepLink, openInNewTab, isExternalLink, renderItem } = item;
+  const { id, path, icon, deepLink, openInNewTab, isExternalLink, renderItem } = item;
 
   const href = deepLink?.url ?? item.href;
 
-  const onClick = useCallback<React.MouseEventHandler>(
+  const onClick = useCallback<React.MouseEventHandler<HTMLButtonElement | HTMLElement>>(
```

Comment:
nit: is this the right type?  `EuiListGroupItem` has a [`HTMLButtonElement`](https://github.com/elastic/eui/blob/3a779b1b8d25bcd1eadb5bea9bcb985794e241c1/packages/eui/src/components/list_group/list_group_item.tsx#L131) onclick.